CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+# SPDX-FileCopyrightText: 2009 Cristian Oneț <onet.cristian@gmail.com>
```

Comment:
libofx doesn't use SPDX, nor does an SPDX line comply with the requirements of copyright law or the terms of the Gnu General Public License.  See https://www.gnu.org/licenses/gpl-howto.html for instructions on how to correctly document the copyright and GPL licensing in files.

Prior to [your changes](https://github.com/KDE/kmymoney/commit/09172a150bd2185889b9887a4406a6374c0726a0) the copyright and license of this file was
```
# Copyright (c) 2006, Alexander Neundorf, <neundorf@kde.org>
# Copied from FindLibXslt.cmake, 2009, Guillaume DE BURE, <gdebure@yahoo.com>
#
# Redistribution and use is allowed according to the terms of the BSD license.
# For details see the accompanying COPYING-CMAKE-SCRIPTS file.
```
Aside from screwing up the copyright claim in that change you discarded the license statement, violating Alexander Neudorf's terms. Contrary to your claim in the commit message, this file was not created by Christian Onet, he just lifted it from somewhere else and committed it to KMyMoney.

Reply:
I spent a good chunk of time to dig down to the first author of the file, which I actually first noticed used as part of a patch here: https://invent.kde.org/packaging/craft-blueprints-kde/-/blob/master/libs/libofx/libofx.py

I eventually stumbled upon the same code in KMyMoney, but actually failed to notice the header mentions the name author different than that of the committer. Do note that time I was working with the `craft-blueprints-kde`'s copy of the file from the beginning, which already had had the license header removed, so I simply assumed Christian was the author himself.

So, while I appreciate pointing this out, jumping into conclusion and implying I intentionally discarded the license and violated author's rights is a bit far. I mean, we all (at least I do) it for free, so what potential gain would I have from changing the ownership of a 15 y/o file from one 3rd party to another 3rd party?

Reply:
Also,

> Contrary to your claim in the commit message, this file was not created by Christian Onet, he just lifted it from somewhere else and committed it to KMyMoney.

Christian has also subsequently amended the file himself, so absolutely has right to be mentioned there. 

Lastly, I rewrote `FindOpenSP` module from scratch, so the license no longer applies there.